src/db/nedb.js

```diff
@@ -57,13 +63,26 @@ class DBHelper {
     }
   }
 
-  static async insertOrUpdateCOracle(db, oracle) {
+  static async removeTopic(db, txid) {
     try {
+      const numRemoved = await db.remove({ txid }, {});
```

Comment:
` db.remove({ txid }, { multi: true })`, or it will always return the `1`

Reply:
fixed. curious, why do we need to set multi when there should only be a single entity?